CONTRIBUTING.md

```diff
@@ -51,10 +51,6 @@ For more information see the [Code of Conduct FAQ](https://aws.github.io/code-of
 opensource-codeofconduct@amazon.com with any additional questions or comments.
 
 
-## Security issue notifications
```

Comment:
Why remove this rather than update it? It seems far more discoverable than the GitHub security policy page.